static/js/subs.js

```diff
@@ -253,7 +254,6 @@ exports.update_settings_for_unsubscribed = function (sub) {
 
         // If user unsubscribed from private stream then user can not subscribe to
         // stream without invitation. So hide subscribe button.
-        stream_data.update_calculated_fields(sub);
```

Comment:
The call to `stream_data.update_calculated_fields` really belongs in another part of the code entirely; fixed in c5904499db2d6a47da9e48eea743b26f38b64003